README.md

```diff
@@ -50,11 +50,11 @@ To run the hooks over Jupyter Notebooks too, add `jupyter` to the list of allowe
   hooks:
     # Run the linter.
     - id: ruff
-      types_or: [ python, pyi, jupyter ]
+      types_or: [ python, pyi ]
```

Comment:
`types_or` _overwrites_ the default config in `.pre-commit-hooks.yaml` rather than _extending_ it, correct? I think that's what we want, I'm just double-checking.

Reply:
Yes, that's how everything in pre-commit works. Users can override anything.